core/amber/src/main/scala/edu/uci/ics/texera/web/WebUtils.scala

```diff
@@ -6,9 +6,22 @@ import java.net.URL
 import akka.actor.{ActorSystem, Props}
 import com.typesafe.config.ConfigFactory
 import edu.uci.ics.amber.clustering.ClusterListener
+import edu.uci.ics.texera.workflow.common.Utils
 
 object WebUtils {
 
+  var googleTokenPath: String = "";
```

Comment:
toggle loading of google token with `environment.downloadExecutionResultEnabled`

Reply:
`environment.downloadExecutionResultEnabled` is frontend configuration so I believe it shouldn't be using here.

Reply:
Correct. The point is to make it work without google configs: developers don't usually have google config files